simplefs/simplefs_test.go

```diff
@@ -102,6 +102,9 @@ func checkPendingOp(ctx context.Context,
 	case keybase1.AsyncOps_LIST_RECURSIVE:
 		list := o.ListRecursive()
 		assert.Equal(t, list.Path, src, "Expected matching path in operation")
+	case keybase1.AsyncOps_LIST_RECURSIVE_TO_DEPTH:
```

Comment:
This case won't actually matter until there's a test that calls the `ToDepth` variant, right?  Might be nice if this PR added something like that.  (I know the regular recursive call tests the code with -1, but would be good to have something that stops early.

Reply:
Will do. We don't currently have a test for `ListRecursive` either, so I'll add one.